core/src/main/java/de/flo56958/MineTinker/Modifiers/Types/SpidersBane.java

```diff
@@ -0,0 +1,154 @@
+package de.flo56958.MineTinker.Modifiers.Types;
+
+import de.flo56958.MineTinker.Data.ModifierFailCause;
+import de.flo56958.MineTinker.Data.ToolType;
+import de.flo56958.MineTinker.Events.ModifierFailEvent;
+import de.flo56958.MineTinker.Main;
+import de.flo56958.MineTinker.Modifiers.Craftable;
+import de.flo56958.MineTinker.Modifiers.Modifier;
+import de.flo56958.MineTinker.Utilities.ChatWriter;
+import de.flo56958.MineTinker.Utilities.ConfigurationManager;
+import de.flo56958.MineTinker.Utilities.Modifiers_Config;
+import org.bukkit.Material;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemFlag;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SpidersBane extends Modifier implements Craftable {
+
+    private boolean compatibleWithSmite;
+    private boolean compatibleWithSharpness;
+
+    private static SpidersBane instance;
+
+    public static SpidersBane instance() {
+        synchronized (SpidersBane.class) {
+            if (instance == null) instance = new SpidersBane();
+        }
+        return instance;
+    }
+
+    private SpidersBane() {
+        super(ModifierType.SPIDERSBANE,
+                new ArrayList<>(Arrays.asList(ToolType.AXE, ToolType.SWORD)),
+                Main.getPlugin());
+    }
+
+    @Override
+    public List<Enchantment> getAppliedEnchantments() {
+        List<Enchantment> enchantments = new ArrayList<>();
+        enchantments.add(Enchantment.DAMAGE_ARTHROPODS);
+
+        return enchantments;
+    }
+
+    @Override
+    public void reload() {
+        FileConfiguration config = getConfig();
+        config.options().copyDefaults(true);
+
+        String key = "SpidersBane";
```

Comment:
In my opinion it looks better on the tool if `SpidersBane` is written like `Spiders-Bane`. (does not need to be the key, but maybe the name value)

Reply:
Is the key what's displayed on the tool in its lore?
If so, it may be worth it to separate the display name and the key.
Adding apostrophes to keys doesn't sound very good to me.

Reply:
In this instance yes; just change the var key used in the next line to the name

Reply:
Oh I see what you mean